timeline/static/js/src/timeline.js

```diff
@@ -1,40 +1,109 @@
 /* Javascript for TimelineXBlock. */
 function TimelineXBlock(runtime, element) {
     const uniqueId = $(element).find('.timeline-container').data('unique-id');
+    const container = $(element).find(`#timeline-${uniqueId}`);
+    const helpPanel = $(element).find(`#help-text-${uniqueId}`);
+    const helpButton = $(element).find(`#help-btn-${uniqueId}`);
+    let timeline;
+
+    helpButton.on("click", () => {
+        helpPanel.toggle();
+        helpButton.attr('aria-expanded', helpPanel.is(':visible'));
+    });
+
+    const handleKeys = (item = null) => (event) => {
+        if (!timeline) return;
+        switch (event.key) {
+            case 'Space':
```

Comment:
```suggestion
            case ' ':
```

`'Space'` doesn't work for me, but literal `' '` does.